src/ui/Button.jsx

```diff
@@ -0,0 +1,24 @@
+import styles from "./Button.module.css";
+
+const Button = ({
+  type = "button",
+  variant,
+  size,
+  children,
+  className = "",
+  onClick,
+}) => {
+  const btnVariant = `btn-${variant}`;
+  const btnSize = `btn-${size}`;
+  return (
+    <button
+      type={type}
+      className={`${className} ${styles[btnVariant]} ${styles[btnSize]}`}
```

Comment:
여러개의 클래스이름을 사용할때 더 나중 순서에 사용되는 클래스이름에 의해 스타일이 재정의될 수있으니, props로 받은 className을 아래와 같이 더 나중 순서로 바꿔 사용해주시는게 의도에 맞는 사용입니다 :)

```suggestion
      className={`${styles[btnVariant]} ${styles[btnSize]} ${className}`}
```